### ISSUE TITLE
Improve selecting regional parameters versions in the Console

### ISSUE THREAD

**Issue by bafonins** [TheThingsNetwork/lorawan-stack#2784]
#### Summary

See discussion starting at https://github.com/TheThingsNetwork/lorawan-stack/pull/2644#issuecomment-644146615

#### Why do we need this?

More guidance for the users.

#### What is already there? What do you see now?

Selects for lorawan and regional parameters versions.

#### What is missing? What do you want to see?

Regional parameters version can be aligned with lorawan version.

#### Environment

v3.8.4

#### How do you propose to implement this?

Filter `lorawan_phy_version` field based on selected `lorawan_version`

```
| lorawan version  | lorawan phy version  |
|------------------|----------------------|
|       1.0.0      |        1.0.0         |
|       1.0.1      |        1.0.1         |
|       1.0.2      |   1.0.2-a, 1.0.2-b   |
|       1.0.3      |       1.0.3-a        |
|       1.0.4      |         (?)          |
|       1.1.0      |   1.1.0-a, 1.1.0-b   |
```

cc @johanstokking @laurensslats What about `1.0.4` ?

#### How do you propose to test this?

Check different selections of `lorawan_version` in the console (both the create and general settings forms)

#### Can you do this yourself and submit a Pull Request?

yes

**Comment by johanstokking** [comment]
Yeah please take a seat and grab a tissue because here comes the tear-jerking story.

The Technical Committee used three different versioning schemes so far: TS001 (LoRaWAN specification), RP001 ("Regional Parameters") and RP002 ("RP2 Regional Parameters").

To make matters worse, the latter versioning scheme is introduced after 1.1 got published, and before 1.0.4 got published. Nobody outside the LoRa Alliance understands this, but we have to deal with this.

To make matters even worse, the "revision" specifier got added after RP001 1.0.2, so we have 1.0.2 and 1.0.2 rev. B, but there's no official 1.0.2 rev. A. So device makers can say they use "LoRaWAN 1.0.2" but we can't blame them for not knowing which PHY version they use.

So:

LoRaWAN version | Regional Parameter specification | Versions | My proposal
--- | --- | --- | ---
1.0 | TS001 | n/a | 1.0
1.0.1 | TS001 | n/a | 1.0.1
1.0.2 | RP001 | 1.0.2, 1.0.2 rev. B | 1.0.2 rev. A, 1.0.2 rev. B
1.0.3 | RP001 | 1.0.3 rev. A | 1.0.3 rev. A
1.0.4 | RP002 | 1.0.0, 1.0.1 | RP2 1.0.0, RP2 1.0.1
1.1 | RP001 | 1.1 rev. A, 1.1 rev. B | 1.1 rev. A, 1.1 rev. B
1.1.1 * | RP002 | 1.0.0, 1.0.1 | RP2 1.0.0, RP2 1.0.1

\* don't list this version yet, illustrative only

**Comment by ElectronicallyE** [comment]
![Image](https://github.com/user-attachments/assets/e8b2112d-7e70-4fd4-8dbf-95a4b0cabb58)
